src/scripts/main.js

```diff
@@ -165,8 +163,8 @@ window.onload = function() {
 
 		// Reset carousel countdown
 		window.removeEventListener('carousel-content-visible', onCarouselVisibleShowCountdown);
-		carouselCountdown.style.transform = 'scaleX(0)';
 		carouselCountdown.style.transition = 'none';
+		carouselCountdown.style.transform = 'scaleX(0)';
```

Comment:
I wanted that transform to occur after the transition had been removed, that way, there's no chance of the bar shooting from one end of the screen to another if a value somehow becomes assigned to it.